public/app/plugins/datasource/prometheus/querybuilder/shared/QueryBuilderHints.tsx

```diff
@@ -56,7 +56,7 @@ export const QueryBuilderHints = <T extends PromLokiVisualQuery>({
                   size="sm"
                   className={styles.hint}
                 >
-                  {'hint: ' + hint.fix?.action?.type.toLowerCase().replace('_', ' ') + '()'}
+                  {'hint: ' + hint.fix?.action?.type.toLowerCase().split('_').join(' ')}
```

Comment:
This is a good approach, as it improves the output we display, but I think we can go one step further, and craft a suggestion message that is not the transformed name of the action.

For that, I would add a `hint` property (label has already been (mis)used) property to `action`, and use it here.

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/queryHints.ts#L29

Since this is a shared component, we may want to either also improve it for Prometheus, or use `.hint` in Loki and fallback to `hint.fix?.action?.type.toLowerCase().split('_').join(' ')` for Prometheus. Your call.